.github/workflows/llvm-project.yml

```diff
@@ -0,0 +1,31 @@
+name: Build kernel.org toolchain
+
+on: workflow_dispatch
+jobs:
+  stage_1:
+    name: stage 1
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        image: [llvm-project]
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build
+        uses: docker/build-push-action@v2
+        with:
+          context: ./${{ matrix.image }}
+          file: ./${{ matrix.image }}/Dockerfile
+          platforms: linux/amd64
+          push: ${{ github.repository_owner == 'ClangBuiltLinux' && github.event_name != 'pull_request' && github.ref_name == 'main' }}
+          tags: |
+            ghcr.io/clangbuiltlinux/${{ matrix.image }}:latest
```

Comment:
Do we want to tag this as an intermediate container image, like `stage2` instead of `latest`?

Reply:
done PTAL